cmake/Codegen.cmake

```diff
@@ -76,11 +76,15 @@ function(GEN_XPU file_yaml)
     --per-operator-headers
     --static-dispatch-backend
     --backend-whitelist=XPU
+    --xpu
+    --update-aoti-c-shim
+    --aoti-extend
+    --aoti-install-dir ${XPU_AOTI_INSTALL_DIR}
```

Comment:
Are the two parameters newly added?

Reply:
--aoti-extend is add in https://github.com/pytorch/pytorch/pull/136742